doc/src/learn/architecture/consensus.md

```diff
@@ -75,15 +78,15 @@ flowchart TB
 * The certificates prove the data availability of each collection, or block, at each round.
 * Their contents constitute a DAG that can be traversed identically at each honest node.
 
-While the Bullshark consensus selects a specific DAG traversal among several a posteriori, both Bullshark and external consensus algorithms can add more sophistication to their selection of blocks / collections to reflect priority concerns.
+While the Bullshark or Tusk consensus selects a specific DAG traversal among several a posteriori, both they and external consensus algorithms can add more sophistication to their selection of blocks / collections to reflect priority concerns.
```

Comment:
algorithms can add

could be more clear to say that they support

what does "more sophistication" mean here? Is it trying to say that they support a more sophisticated selection of blocks based on priority? I am not clear on what the benefit is

Reply:
This text wasn't altered during this PR, so I would once again recommend you take up wholesale edits and reviews in distinct issues and PRs.

Still, since @huitseeker wrote this, I will keep it open for now to see if he thinks this needs clarification.

Reply:
> what does "more sophistication" mean here? Is it trying to say that they support a more sophisticated selection of blocks based on priority?

Yep, that's exactly what this is reaching for.